policy/release/tasks.rego

```diff
@@ -100,3 +97,26 @@ _task_names(task, raw_name) = names if {
 
 	names := {name} | params
 }
+
+# The latest set of required tasks. Tasks here are not required right now
+# but will be required in the future.
+_latest_required_tasks contains task if {
+	some task in data["required-tasks"][0].tasks
```

Comment:
Should we sort in case `data["required-tasks"]` is not sorted by date?

Reply:
Good question. So far we don't do any sorting because we rely on ec-cli to do the right thing. That's a good idea though. I'll add it to a follow up PR and file a story to do the same for acceptable bundles.